src/vs/workbench/services/extensions/electron-browser/extensionService.ts

```diff
@@ -313,6 +313,12 @@ export class ExtensionService extends Disposable implements IExtensionService {
 			}
 		}
 
+		// {{SQL CARBON EDIT}}
+		// Forces a reload for CMS extension
+		if (extension.name === 'cms') {
```

Comment:
can't the extension do something to declare it needs a reload?  I haven't investigate myself, but it seems unusual to need to hard code this in the core extension management code.